src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -226,8 +226,14 @@ public class Thread implements Runnable {
         registerNatives();
     }
 
-    /* Reserved for exclusive use by the JVM, maybe move to FieldHolder */
-    private long eetop;
+    /* Reserved for exclusive use by the JVM. Cannot be moved to the FieldHolder
+       as it needs to be set by the VM before executing the constructor that
+       will create the FieldHolder. The historically named `eetop` holds the address
+       of the underlying VM JavaThread, and is set to non-zero when the thread is started,
+       and reset to zero when the thread terminates. A non-zero value indicates this thread
+       isAlive().
+    */
```

Comment:
Maybe JavaDocify this?
```suggestion
    /**
     * Reserved for exclusive use by the JVM. Cannot be moved to the FieldHolder
     * as it needs to be set by the VM before executing the constructor that
     * will create the FieldHolder. The historically named `eetop` holds the address
     * of the underlying VM JavaThread, and is set to non-zero when the thread
     * is started, and reset to zero when the thread terminates. A non-zero value
     * indicates this thread isAlive().
     */
```

Reply:
Being a private field it won't appear in any javadoc that anyone is ever likely to read. Only developers will see this (or need to see it) when they read the source code.

Reply:
It's just a bit inconsistent with the other comments, no big deal.  I'd probably put an asterisk at the bringing of each time so it's consistent with the other /*..*/ comments on fields in this class.

Reply:
Added the asterisks. Thanks